app/views/hyrax/base/_work_title.erb

```diff
@@ -1,9 +1,16 @@
 <% presenter.title.each_with_index do |title, index| %>
-    <% if index == 0 %>
-        <h1>
-            <%= title %> <%= presenter.permission_badge %> <%= presenter.workflow.badge %>
-        </h1>
-    <% else %>
-        <h1><%= title %></h1>
-    <% end %>
+  <div class="row">
+    <div class="col-sm-8">
+      <% if index == 0 %>
+        <h4><%= title %>
+        <%= presenter.permission_badge %> <%= presenter.workflow.badge %>
+      </h4>
+      <% else %>
+          <h1><%= title %></h1>
+      <% end %>
+    </div>
+    <div class="col-sm-4 text-right">
+      <%= render "show_actions", presenter: @presenter %>
```

Comment:
Since we're using presenter as a local variable above can we use the local here too rather than the ivar?

Reply:
Yeah, that was just a copy/paste from the show page when reformatting. Will fix.